### ISSUE TITLE
New data are adding to end of serviceData

### ISSUE THREAD

**Issue by XHunter74** [AlCalzone/ioBroker.ble#779]
After re-install BLE adapter sensor data stopped refreshing - in states adapter saved just first data after start. Re-start adapter - and I have data on this moment. After enabling debug, I detected that new data are added to the end of serviceData array. I have only one BLE sensor so it easy to see this behavior in the log.

**Versions:**  
 - Adapter version: 0.13.0
 - JS-Controller version: 4.0.24
 - Node version: v14.21.3
 - Operating system: Ubuntu 18.0.4

[error.log](https://github.com/AlCalzone/ioBroker.ble/files/10818120/error.log)

**Comment by AlCalzone** [comment]
duplicate of https://github.com/AlCalzone/ioBroker.ble/issues/778, blocked by https://github.com/abandonware/noble/issues/296
